lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -88,4 +88,4 @@ paramiko==2.2.1
 ecdsa==0.13
 
 # GenomeSpace dependencies
-python-genomespaceclient==0.1.8
+python-genomespaceclient>=1.0.0,<2.0.0
```

Comment:
This should be an exact version, i.e. `==1.0.0`

Reply:
@nsoranzo For replicability? Will fix.

Reply:
Yes, we want to freeze the virtualenv for releases. Thanks!